yt_dlp/__init__.py

```diff
@@ -909,6 +915,7 @@ def parse_options(argv=None):
         'postprocessors': postprocessors,
         'fixup': opts.fixup,
         'source_address': opts.source_address,
+        'impersonate': opts.impersonate,
```

Comment:
Even if we don't use namedtuple for YoutubeDL, we should definitely use it at this point